docs/QUARANTINE_GUIDELINES.md

```diff
@@ -0,0 +1,218 @@
+Make sure to:
+- Be specific about what failed and why
+- Avoid generic messages like "Test failed" or "Assertion error"
+- If a product bug was opened, link to the bug in the analysis message
+
+#### Category 1: Product Bug
+
+**Identification**: Failure is due to an actual product defect.
+
+**Actions:**
+- Open a bug in Jira with the appropriate priority (and a fix version if not opened on the next y-stream release).
+- Use `pytest_jira` integration to skip test conditionally when bug is open.  If needed, use `is_jira_open` for conditional skip.
+
+Conditional skip for product bug:
+
+```python
+import pytest
+
+@pytest.mark.jira("CNV-12345", run=False)  # Skips if the bug is open
+def test_feature_a():
+    pass
+```
+
+#### Category 2: Automation Issue
+
+**Identification**: Failure is due to test code or test framework issue.
+
+**Actions:**
+- Open a task in Jira under the team's backlog
+- **Manual Verification MANDATORY**: Verify that the scenario passes manually before marking analysis as complete
+  Manual verification is needed to make sure the tested feature does work as expected before quarantining the test.
+- Include ALL failure analysis information and logs in Jira
+
+Quarantine for automation issue:
+
+Apply pytest's `xfail` marker with `run=False`, for example:
+
+```python
+import pytest
+from utilities.constants import QUARANTINED
+
+
+@pytest.mark.xfail(
+    reason=f"{QUARANTINED}: VM is going into running state which it shouldn't, CNV-123",
+    run=False,
+)
+def test_my_failing_test():
+    ...
+```
+
+**Benefit**: Provides pytest summary insights:
+```
+1 deselected, 2 passed, 1 quarantined, ...
+```
+
+**Jira Requirements:**
+- Title starts with `[stabilization]`
+- Labels: `quarantined-test`
+- Priority based on test importance
+- Complete failure analysis documentation
+- If backport is needed, make sure to add the info in the ticket
+
+#### Category 3: System / Environment Issue
+
+**Identification**: Failure is due to infrastructure, environment, or external dependencies.
+
+**Actions:**
+- **CANNOT QUARANTINE** - Open a ticket to devops
+- **Manual Verification MANDATORY**: Verify that the scenario passes manually before marking analysis as complete
+- Perform additional analysis
+
+**Additional Analysis Required:**
+1. Can the failure be avoided by running test on specific hardware?
+2. Can the failure be caught during cluster sanity checks?
+3. Open an issue to update cluster sanity to include environment requirements that prevent this failure
+
+**System Issue Examples:**
+- Artifactory connection problems
+- Network infrastructure issues
+- Resource constraints
+
+### Step 3: Quarantine Decision
+
+Determine if a test requires quarantine is based on:
+- Failure is not a product bug
+- Repeated failures (twice or more in the past 10 runs)
+- Root cause requiring extended analysis
+- Tests should be migrated between tiers (e.g., tier2 → tier1)
```

Comment:
what's the point to quarantine in this case? just drop it in t2 and create pr/task for t1?